panda/html/signup.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="kor">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="../img/randing/favicon.ico">
+    <link rel="stylesheet" href="../css/signup.css">
+    <link href="https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff">
+    <title>판다마켓-회원가입</title>
+</head>
+<body>
+    <main>
+        <a class="home" href="../html/index.html">
+            <img src="../img/login/home.png" alt="홈 바로가기">
+        </a>
+        <form>
+            <div class="form">
+                <label class="input-label" for="email">이메일</label>
+                <input class="input" id="email" type="email" placeholder="이메일을 입력해 주세요" required>
+            </div>
+
+            <div class="form">
+                <label class="input-label" for="nick">닉네임</label>
+                <input class="input" id="nick" type="text" placeholder="닉네임을 입력해 주세요" required>
+            </div>
+
+            <div class="form">
+                <label class="input-label" for="password">비밀번호</label>
+                <div  class="pw-input">
+                    <input class="input" id="password" type="password" placeholder="비밀번호를 입력해 주세요" required>
+                    <button type="button" class="password-toggle-btn">
+                        <img class="password-toggle" src="../img/login/pasword.png">
+                    </button>
+                </div>
+            </div>
+
+            <div class="form">
+                <label class="input-label" for="passwordchk">비밀번호 확인</label>
+                <div  class="pwchk-input">
+                    <input class="input" id="passwordchk" type="password" placeholder="비밀번호를 다시 한 번 입력해 주세요" required>
+                    <button type="button" class="password-toggle-btn">
+                        <img class="password-toggle" src="../img/login/pasword.png">
+                    </button>
+                </div>
+            </div>
+
+            <div class="signup-btn-container">
+                <button class="signup-btn" type="button">회원가입</button>
```

Comment:
`form`안에서 `button`은 `type:submit`이어야 합니다. 여러버튼이 있다면 제출버튼의 역할을 하는 버튼이 submit 버튼이어야합니다. 


```suggestion
                <button class="signup-btn" type="submit">회원가입</button>
```